configs/sigdep3/identifier_update.sql

```diff
@@ -0,0 +1,33 @@
+USE openmrs;
```

Comment:
This file needs to be run automatically after the DB is initialized the first time (and all of the FHIR2 tables are created)